CONTRIBUTORS.md

```diff
@@ -7,26 +7,33 @@ Chapel release:
 Contributors to the current release
 -----------------------------------
 * Ben Albrecht, [Cray Inc.]
-* Sidhant Bendre, individual contributor
+* Akshansh Bhanjana, individual contributor
 * Paul Cassella, [Cray Inc.]
 * [Brad Chamberlain], [Cray Inc.]
-* Naman Chikara, individual contributor
-* Krishna Kumar Dey, individual contributor
+* Sanket Chaudhari, individual contributor
+* Anway De, individual contributor
+* Garvit Dewan, [GSoC 2019] student from [Indian Institute of Technology, Roorkee]
```

Comment:
Hi, can you remove the `,` between Technology and Roorkee? The name of my school is Indian Institute of Technology Roorkee (without the `,`).

Reply:
Yep, no problem.